app/src/main/java/com/ftw/hometerview/ui/onboarding/OnboardingActivity.kt

```diff
@@ -0,0 +1,40 @@
+package com.ftw.hometerview.ui.onboarding
+
+import android.content.Context
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.inputmethod.EditorInfo
+import com.ftw.hometerview.databinding.ActivityOnboardingBinding
+import com.ftw.hometerview.ui.onboardingresult.OnboardingResultActivity
+
+class OnboardingActivity : AppCompatActivity() {
+
+    companion object {
+        fun newIntent(context: Context): Intent {
+            return Intent(context, OnboardingActivity::class.java)
+        }
+    }
+
+    private lateinit var binding: ActivityOnboardingBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        binding = ActivityOnboardingBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+        
+        binding.searchButton.setOnEditorActionListener { v, actionId, event ->
+            if(actionId == EditorInfo.IME_ACTION_SEARCH){
+                // 보내기 동작
+                startActivity(
+                    OnboardingResultActivity.newIntent(
+                        this, binding.searchButton.text.toString()
+                    )
+                )
```

Comment:
newIntent() 의 parameter 명과 동일한 값이 아니면 앞에 정의해주는 게 가독성에 도움이 될 것 같아!

```suggestion
                    OnboardingResultActivity.newIntent(
                        context = this, 
                        result = binding.searchButton.text.toString()
                    )
                )
```